src/Aspire.Dashboard/Aspire.Dashboard.csproj

```diff
@@ -15,7 +15,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Protobuf Include="**/*.proto" GrpcServices="Server">
+    <!-- Build service and client types. Integration tests use the client types to call OTLP services. -->
```

Comment:
The generated code is visible because of internals visible to and there are shared types that would conflict. I looked into it and there is no way to change the namespace from the command line.

It's not ideal but there won't be any negative impact. The generated code won't get used, and if we ever trim/AOT the app then it would get removed.